components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -221,4 +371,105 @@ Should you have a need to revert back to IAM v1, it is quick and easy to do:
 $ chef-automate iam reset-to-v1
 ```
 
-Note that this discards your IAM v2 policies and roles, and re-configures Chef Automate to use your v1 policies again.
+Note that this discards your IAM v2 policies, roles, and projects, and re-configures Chef Automate to use your v1 policies again.
+
+<!-- TODO move this next to the rest of the projects walkthrough -->
+## Assigning Projects to Ingested Resources
+
+While resources created within Automate can be directly added to a project, ingested resources, including Compliance reports, events, and nodes, are created outside of Automate and therefore must be added to projects a different way.
+
+Project rules are used to associate ingested resources with projects within Automate. A project rule contains conditions that determine if an ingested resource should be moved into the rule's project.
+Each condition contains an attribute, operator, and value.
+
+{{% info %}}
+See the [API reference]({{< relref "iam-v2-api-reference.md#project-rules" >}}) for details on how to manage project rules.
+{{% /info %}}
+
+In this example you will add a project rule to the project `project-devops`, created as part of the [project creation walkthrough]({{< relref "iam-v2-guide.md#creating-a-project" >}}).
```

Comment:
it be cool in the link text could match the header exactly, _creating a project_.. so either change the link text or the heading